backend-node/webhooks/rtc-events/image.js

```diff
@@ -0,0 +1,42 @@
+const Data = require('../../data');
+
+async function create(client, reqBody) {
+  const { type, conversation_id, to, from, body, id, timestamp } = reqBody;
+  console.log(JSON.stringify(conversation_id));
+  if(!type || type != 'image' || !conversation_id || !from || !body || !id || !timestamp) {
+    return 'Missing data';
+  }
+  const { url } = body.medium.url;
+  console.log(`URL: ${JSON.stringify(url)}`);
+
+  if(!url ) {
+    return 'Missing data - body/medium/url';
+  }
+
+  let conversation = await Data.conversations.get(client, conversation_id);
+  if(!conversation) {
+    return 'Could not find the conversation';
+  }
+
+  let fromMember = await Data.members.get(client, from);
+  if(to) {
+    let toMember = await Data.members.get(client, to);
+  }
+  if(!fromMember) {
+    // TODO - sync not complete
+    await Data.conversations.syncMembers(client, conversation_id);
+  }
+  // TODO - recheck the fromMember
+
+  let event = await Data.events.create(client, id, type, conversation_id, from, to, url, timestamp);
+  if(event) {
+    return `Created text event: ${JSON.stringify(event)}`;
```

Comment:
Should this be "Created image event". Making sure I understand what's happening.
```suggestion
    return `Created image event: ${JSON.stringify(event)}`;
```